google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -300,7 +302,7 @@ public GapicSpannerRpc(final SpannerOptions options) {
 
                 // Set a keepalive time of 120 seconds to help long running
                 // commit GRPC calls succeed
-                .setKeepAliveTime(Duration.ofSeconds(GRPC_KEEPALIVE_SECONDS))
+                .setKeepAliveTime(Duration.ofSeconds(GRPC_KEEPALIVE_SECONDS * 1000))
```

Comment:
Did you really intend to set the keepalive time to be over a day? This is a value of 120,000 seconds.

Reply:
Oops. Thanks for noticing. Nope, that's a typical seconds/milliseconds mistake.

Reply:
https://github.com/googleapis/java-spanner/pull/339